cfme/tests/intelligence/reports/test_reports_schedules.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+import pytest
+import yaml
+
+from cfme import test_requirements
+from cfme.utils.path import data_path
+
+schedules_report_dir = data_path.join("schedules_report")
+
+
+def schedule_files():
+    result = []
+    for file_name in schedules_report_dir.listdir():
+        if file_name.isfile() and file_name.basename.endswith(".yaml"):
+            result.append(file_name.basename)
+    return result
+
+
+@pytest.fixture(params=schedule_files())
+def schedule_data(request):
+    with schedules_report_dir.join(request.param).open(mode="r") as rep_yaml:
+        return yaml.load(rep_yaml)
+
+
+@pytest.mark.sauce
+@pytest.mark.tier(3)
+@test_requirements.report
+def test_schedule(schedule_data, appliance):
+    """ To test scheduling of report using options: Once, Hourly, Daily, Weekly, Monthly """
+    schedules = appliance.collections.schedules
+    schedule = schedules.create(**schedule_data)
```

Comment:
**REQUIRED**
This should go to a fixture.

Reply:
@niyazRedhat Let me insist here. You don't test the schedule creation but queuing. Moreover, the schedule deletion should be performed in the teardown.